time_manager/get_cal_time.F90

```diff
@@ -56,291 +55,18 @@ module get_cal_time_mod
 namelist / get_cal_time_nml / allow_calendar_conversion
 ! </NAMELIST>
 
+interface get_cal_time
+  module procedure get_cal_time_r4
+  module procedure get_cal_time_r8
+end interface
+
+public :: get_cal_time
```

Comment:
Why make this public underneath interface?

Reply:
I don't think i moved that on purpose, but the order doesn't matter for public statements.